alert-policies/flutter-alert-policy/highLatency.yml

```
@@ -0,0 +1,27 @@
+name: High Latency
+
+description: |+
+  This alert is triggered when latency is greater than 5s for 1 minute.
+type: STATIC
+nrql:
+  query:
+  "SELECT average(networkLatency) as 'AVG' FROM Flutter FACET CASES(
+  WHERE appName = 'YOUR IOS APP NAME' AS 'iOS',
+  WHERE appName = 'YOUR ANDROID APP NAME' AS 'Android'
+))"
```

This should clear up those yaml errors
```suggestion
  query:
    "SELECT average(networkLatency) as 'AVG' FROM Flutter FACET CASES(WHERE appName = 'YOUR IOS APP NAME' AS 'iOS', WHERE appName = 'YOUR ANDROID APP NAME' AS 'Android'))"
```